pyuvdata/uvbeam.py

```diff
@@ -150,7 +150,9 @@ def __init__(self):
                 'Peak normalized means that for each frequency the data_array'
                 'is separately normalized such that the peak is 1 (so the beam '
                 'is dimensionless) and all frequency dependence is moved to the '
```

Comment:
That's true, but it is common in radio astronomy to split out the direction independent frequency dependence (sorry that's awkward wording) into what is called a bandpass and to therefore have the beam peak normalized at every frequency. That is what I was trying to describe here, I'll see if I can make the language more clear. The main point was that this was different than the physical normalization, where the direction independent frequency dependence is split between the bandpass and the data array based on the source of that frequency dependence.